File: js/stack.js
class Stack {
    constructor() {
        this.items = [];
    }
    add(data) {
        this.items.push(data);
    }
    pop() {
        this.items.pop();
    }
    top() {
        return this.items[this.items.length-1];
    }
}

s = new Stack();
s.add(3);
s.add(5);
s.pop();
s.add(7);
console.log(s.top());
s.add(12);

File: py/stack.py
class Stack:
    def __init__(self):
        self.items = []
        self.size = 0
    
    def add(self, data):
        self.items.append(data)
        self.size += 1

    def pop(self):
        self.items.pop()
        self.size -= 1
    
    def getSize(self):
        return self.size

s = Stack()

s.add(3)
s.add(2)

s.pop()
s.pop()